radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/TokenState.java

```diff
@@ -112,10 +112,11 @@ public boolean equals(Object o) {
 
 	@Override
 	public String toString() {
-		return String.format("Token(%s) name(%s) description(%s) totalSupply(%s) granularity(%s) maxSupply(%s)",
+		return String.format("Token(%s) name(%s) description(%s) url(%s) totalSupply(%s)granularity(%s) maxSupply(%s)",
```

Comment:
Is there a missing space `totalSupply(%s)` and `granularity(%s)`?

Reply:
Yes!